processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/EntityImplementationGenerator.java

```diff
@@ -0,0 +1,250 @@
+package org.mobilitydata.gtfsvalidator.processor;
+
+import com.squareup.javapoet.*;
+import org.mobilitydata.gtfsvalidator.annotation.FieldTypeEnum;
+import org.mobilitydata.gtfsvalidator.annotation.Generated;
+import org.mobilitydata.gtfsvalidator.table.GtfsEntity;
+import org.mobilitydata.gtfsvalidator.type.GtfsColor;
+import org.mobilitydata.gtfsvalidator.type.GtfsDate;
+import org.mobilitydata.gtfsvalidator.type.GtfsTime;
+
+import javax.lang.model.element.Modifier;
+import javax.lang.model.type.TypeMirror;
+import java.time.ZoneId;
+import java.util.TimeZone;
+
+import static org.mobilitydata.gtfsvalidator.processor.FieldNameConverter.*;
+import static org.mobilitydata.gtfsvalidator.processor.GtfsEntityClasses.TABLE_PACKAGE_NAME;
+
+/**
+ * Generates a class that represent a single parsed row of a GTFS table.
+ *
+ * E.g., GtfsStop class is generated for "stops.txt".
+ */
+public class EntityImplementationGenerator {
+    private static final String CSV_ROW_NUMBER = "csvRowNumber";
+    private final GtfsFileDescriptor fileDescriptor;
+    private final GtfsEntityClasses classNames;
+
+    public EntityImplementationGenerator(GtfsFileDescriptor fileDescriptor) {
+        this.fileDescriptor = fileDescriptor;
+        this.classNames = new GtfsEntityClasses(fileDescriptor);
+    }
+
+    private static String maskForFieldNumber(int fieldNumber) {
+        return "0x" + Integer.toHexString(1 << fieldNumber);
+    }
+
+    private static CodeBlock getDefaultValue(GtfsFieldDescriptor field) {
+        if (field.defaultValue().isPresent()) {
+            String valueString = field.defaultValue().get();
+            switch (field.type()) {
+                case BOOLEAN:
+                    return CodeBlock.of(
+                            Boolean.parseBoolean(valueString) || (Integer.parseInt(valueString) == 1) ?
+                                    "true" : "false");
+                case INTEGER:
+                case ENUM:
+                    return CodeBlock.of(Integer.toString(Integer.parseInt(valueString)));
+                case FLOAT:
+                case LATITUDE:
+                case LONGITUDE:
+                    return CodeBlock.of(Double.toString(Double.parseDouble(valueString)));
+                case COLOR:
+                    return CodeBlock.of("$T.fromInt(0x$L)",
+                            GtfsColor.class, Integer.toHexString(Integer.parseInt(valueString, 16)));
+                case TEXT:
+                    return CodeBlock.of("$S", valueString);
```

Comment:
This would be bug-prone since it is not a trivial task to escape an arbitrary string to a Java string literal.
$S is the placeholder that handles escaping. It will add the double quotes and other things.